src/m1_facilitator/repositories/ERC20GatewayTokenPairRepository.ts

```diff
@@ -0,0 +1,176 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+
+import { DataTypes, Model, InitOptions } from 'sequelize';
+
+import ERC20GatewayTokenPair from '../models/ERC20GatewayTokenPair';
+import Subject from '../../m0_facilitator/observer/Subject';
+import Utils from '../../m0_facilitator/Utils';
+
+import assert = require('assert');
+
+/* eslint-disable class-methods-use-this */
```

Comment:
Otherwise, we should make get and save functions static to eliminate linter error.
I definitely do not like to see them as a static (this happens because of sequelize).

Reply:
@deepesh-kn I am not sure what do you mean here by `convertToGateway` function. Also, I've referred to this point in other comment below. Can you please elaborate more on this?

Reply:
I corrected my comment above.
I meant was, we are already using this eslint disable comment in all the repository files, so there is no problem here with using it `/* eslint-disable class-methods-use-this */`